### PR TITLE
Issue#88 adjust tag line

### LINKED ISSUE #88: Improve home page header text
Currently we have, a thrown together at the time (by me):
**Rockstor now on Pi4 Ten64 & More**

Where-as, given the increasing number of aarch64 SBC, some of which our Ten64 ARM64EFI installers will work on, via the [Embedded Boot](https://github.com/ARM-software/ebbr) or [Server boot](https://github.com/ARM-software/sbsa-acs) standard, it is proposed that we modify this key header to emphasise the ARM64EFI compatibility and not have this over-rule our X86_64 stalwart compatibility.

I'm thinking here of half-brained reviews indicating our CentOS x86_64 only nature from many years ago! - while showing a picture of the Pi4 (in famous case) in a presumably automated screen home page grab.

So we are after something to highlighting our cross arch capability i.e. 'AMD64/ARM64EFI' in line with @Hooverdan96's work to formalise our arch referenced in for example https://github.com/rockstor/rockon-registry/pull/323 , and given the **Ten64** platform folks (via @mcbridematt) have supported the project via hardware donation and code contribution/review it would be nice to maintain the Ten64 highlight here: especially given we have installers custom made for them (again thanks to @mcbridematt) and we feature in their docs:
https://ten64doc.traverse.com.au/applications/doesitrun/#storage-and-nas
https://ten64doc.traverse.com.au/applications/rockstor/

So all in what this looks to boil down to is more explict mention of both architectures while maintaining single line succinctness, and ideally also have for example a 'Uses openSUSE' element in there. Doable with a reduction in conjunction likely.

Many folks will see only pics and heading - and if nothing grabs their eye or they miss-read they may think we are a custom OS with little backing yet appreciate the difficulty of rolling ones own - yet this appreciation is in part a defining characteristic of the DIY NAS audience. So lets update this header a tad to not throw the baby out with the bath-water :).

### PR DESCRIPTION
**Pull request by Hooverdan96**
Fixes #88.

- adjusted tag line
- fixed typo in FAQ for customizable platform section

### CODE REVIEW

**Comment by Hooverdan96** [comment]
Feel free to adjust further!

File: content/features/_index.md [hunk 1]
```diff
@@ -6,9 +6,10 @@ date: 2021-10-15T11:02:11+01:00
 draft: false
 ---
 
-Rockstor is a Linux and BTRFS powered storage operating system with advanced NAS and Cloud Storage features.
+Rockstor is a Linux and BTRFS powered storage operating system with advanced NAS and Cloud Storage features, supporting
+multiple architectures like AMD64/ARM64EFI/Ten64 and more.
```
Comment (phillxnet) [inline content/features/_index.md:10]:
```suggestion
multiple architectures: AMD64 & ARM64EFI.
```
Backing away from the Ten64 mention now given we have Traverse Technologies in our Partners section on the landing page: plus Ten64 is a platform, and an example of ARM64EFI. And we line up our future RISC-V compatibility: an inevitability as we update our base OS and dependencies. I know this looses our Ten64 mention and our Pi4 mention but we have Partner and Pic references prominent, respectively, for both.  Plus we have Pi4 (should add Pi5 once proven) & Ten64 specific downloads; and the Ten64 docs in turn have links back to us.

File: content/solutions/customizable-btrfs-nas-storage-platform.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Watch [this space](https://github.com/rockstor/rockstor-core/issues/2254) on tha
 
 {{< ps phrase="Custom Installer" >}}
 ## Problem
-I/We need more than what Rocksor is; an appliance approach is fine for the majority of cases but not for us.
+I/We need more than what Rockstor is; an appliance approach is fine for the majority of cases but not for us.
```
Comment (phillxnet) [inline content/solutions/customizable-btrfs-nas-storage-platform.md:49]: Thanks, nice fix. Though maybe the original should be been Rocksore. Or maybe Rock-sore-point :) .

**Review by phillxnet — CHANGES_REQUESTED** [review]
@Hooverdan96 Thanks for pushing this one onto a PR.
Made a couple of suggestions - see what you think.

File: content/_index.md [hunk 3]
```diff
@@ -3,4 +3,4 @@ title: "Homepage"
 date: 2021-10-10T11:02:11+01:00
 draft: false
 ---
-## Rockstor now on Pi4 Ten64 & More
\ No newline at end of file
+## An Open Source multi-architecture NAS built on OpenSUSE
```
[the remark this answers is not in the document]
Comment (phillxnet) [reply content/_index.md:6]: Just popping in a side-by-side post my suggestion:
tag-line-pr-side-by-side_post-suggestion

**Comment by Hooverdan96** [comment]
All suggestions accepted, seem to be sensible changes to limit the line spillover and the architecture formulation also makes sense to me.

**Comment by phillxnet** [comment]
PR product PRODUCTION published.

**Pull request merged**